vscode_extension/src/outputChannel.ts

```diff
@@ -0,0 +1,24 @@
+import { window, OutputChannel } from "vscode";
+
+let extensionOutputChannel: OutputChannel;
+
+/**
+ * @returns vscode.OutputChannel for AaC extension output to the VSCode Output Panel.
+ *
+ * @example
+ * Sets the content of an output panel to `Hello!`:
+ * ```ts
+ *  let stringMessage = "Hello!"
+ *  const outputChannel = getOutputChannel();
+ *  outputChannel.appendLine(stringMessage);
+ *  outputChannel.show(true);
+ * ```
+ * @see {@link https://code.visualstudio.com/api/references/vscode-api#OutputChannel | VSCode OutputChannel API documentation}
+ */
+ export function getOutputChannel(): OutputChannel {
+
+    if (!extensionOutputChannel) {
+        extensionOutputChannel = window.createOutputChannel('Architecture-as-Code');
```

Comment:
I just noticed this, but if we use this hard-coded value, then we can cause confusion for users because I called the LSP server output channel the same thing. Maybe we can change the LSP server channel to "Architecture-as-Code Language Server", or something to distinguish them from each other, or we can use this function in the `getClientOptions` function in `AacLanguageServer.ts`.

![Screen Shot 2022-03-01 at 13 00 20](https://user-images.githubusercontent.com/2924081/156231876-40875661-2e5b-4711-9190-b4800513d7f6.png)